batch7crud-roll9/src/main/java/com/lftechnology/batch7crud/controller/StudentControllerServlet.java

```diff
@@ -0,0 +1,153 @@
+package com.lftechnology.batch7crud.controller;
+
+import com.lftechnology.batch7crud.entity.Student;
+import com.lftechnology.batch7crud.exception.DataException;
+import com.lftechnology.batch7crud.services.StudentService;
+
+import javax.servlet.RequestDispatcher;
+import javax.servlet.ServletException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.io.IOException;
+import java.util.List;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+/**
+ * Created by sanjay on 1/18/16.
+ */
+@WebServlet("/students/*")
+public class StudentControllerServlet extends HTTPStatusHandler {
+  private static final Logger LOGGER = Logger.getLogger("appLogger");
+  private static final String ERROR_PAGE = "/WEB-INF/views/error-page.jsp";
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this unused "ERROR_PAGE" private field. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1068)